pyproject.toml

```diff
@@ -10,7 +10,7 @@ requires = [
     # redistributors can do this by installing the numpy version they like and
     # disabling build isolation.
     "numpy>=2.0.0rc1",
```

Comment:
We should probably bump this to be `>=2.0.0` without the rc option.

Reply:
Done :+1: